lessons/lesson_4/js/block_5.js

```diff
@@ -27,3 +27,36 @@ function task_2(month) {
     }
     alert('Пора года : ' + result);
 }
+
+function task_3(str) {
+    (str[0] === 'a') ? alert('Да!') : alert('Нет!');
+}
+
+function task_4(str, number) {
+    if (!isNaN(str)) {
+        (parseInt(str[0]) === number) ? alert('Да!') : alert('Нет!');
+    } else {
+        alert('Строка должна содержать число!');
+    }
+}
+
+function task_5(str) {
+    var numbers = str.split(""),
+        result = 0;
+    if (!isNaN(str) && str.length === 3) {
+        numbers.forEach(function(element){
+            result += parseInt(element);
+        });
+        alert(result);
+    } else {
+        alert('Строка должна содержать 3-х значное число!');
+    }
+}
+
+function task_6(str) {
+    if (!isNaN(str) && str.length === 6) {
+        ((str[0] + str[1] + str[2]) === (str[3] + str[4] + str[5])) ? alert('Да!') : alert('Нет!');
```

Comment:
Move mathematics result to variables and use only variables in the if.
Don't use short if for display alert. Better way for using short if in next case:
`let test = 1 != 2 ? 'yes': 'no'`